plugins/basic/sg_bootstrap.py

```diff
@@ -1,5 +1,5 @@
 """
-The Unity client module for Shotgun. Responsible of customizing the client 
+The Unity bootstrap module for Shotgun. Responsible of customizing the bootstrap
```

Comment:
Please remove the second sentence. It does not make sense now that we moved out of rpyc